app/components/services/disclosure/index.spec.js

```diff
@@ -13,31 +13,76 @@ describe('disclosureApi', function() {
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        localities.each(function(locality) {
+          expect(locality).to.have.property('name');
+          expect(locality).to.have.property('type');
+          expect(locality).to.have.property('id');
+        });
+      });
   });
 
-  it('has locations', function() {
-    expect(disclosureApi).to.have.property('locations');
+  // locality.current_ballot
+  it('get a ballot', function() {
+    // Search for any locality
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        // Get the current ballot for the first locality
+        var locality = localities[0];
+        disclosureApi.locality.current_ballot({locality_id: locality.id})
+          .then(function(ballot) {
+            expect(ballot).to.have.property('locality_id');
+            expect(ballot).to.have.property('ballot_id');
+            expect(ballot).to.have.property('ballot_items');
+          });
+      });
   });
 
-  it('has search', function() {
-    expect(disclosureApi).to.have.property('search');
+  // office_election.get
+  it('get an office election', function() {
+    // Search for any locality
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        // Get the current ballot for the first locality
+        var locality = localities[0];
+        disclosureApi.locality.current_ballot({locality_id: locality.id})
+          .then(function(ballot) {
+            // Get the office election details from the ballot
+            disclosureApi.office_election.get({ballot_id: ballot.ballot_id})
+              .then(function(officeElection) {
+                expect(officeElection).to.have.property('candidates');
+              });
+          });
+      });
   });
 
-  it('lists some contributions', function() {
-    // This hits a live API, so changes outside of this project could fail this test :(
-    return disclosureApi.contributions.list()
-      .then(function(contributions) {
-        expect(contributions[0]).to.have.property('amount');
+  // candidate.get
+  it('get a candidate', function() {
+    // Search for any locality
+    disclosureApi.locality.search({q: ''})
+      .then(function(localities) {
+        // Get the current ballot for the first locality
+        var locality = localities[0];
+        disclosureApi.locality.current_ballot({locality_id: locality.id})
+          .then(function(ballot) {
+            // Get the office election details from the ballot
+            disclosureApi.office_election.get({ballot_id: ballot.ballot_id})
+              .then(function(officeElection) {
+                // Get details from the first candidate.
+                var candidate = officeElection.candidates[0];
+                disclosureApi.candidate.get({candidate_id: candidate.id})
+                  .then(function(candidate) {
+                    expect(candidate).to.have.property('last_name');
```

Comment:
@adborden The API is more granular, so it is necessary to chain API calls to get needed info. I was able to validate the first two tests, but after a while, things didn't seem to be getting through to the backend, and tests seemed to pass regardless of what was inside.

Please take a look.